README.md

```diff
@@ -17,26 +17,20 @@ scripts are included in the package for reference.
 
 ## Compatibility
 
-Supported AEM versions:
+Supported versions:
 
-* On premise: `>= 6.5.10`
+* AEM On premise: `>= 6.5.10`
 * AEM as a Cloud Service: `>= 2022.11`
+* Sling: `>=12`
 
 Embedded Groovy version: `2.4.15`
 
 To install the AEM Groovy Console on older AEM versions check the original
 project [aem-groovy-console](https://github.com/CID15/aem-groovy-console)
 
-## Local installation
+## Installation
 
-### Requirements
-
-* AEM author instance running on [http://localhost:4502](http://localhost:4502/)
-* [Maven](https://maven.apache.org/) `>= 3.6.3`
-
-### Installation
-
-#### Manual
+### Manual
 
 1. Download the
    console [aem-groovy-console-all](https://github.com/orbinson/aem-groovy-console/releases/download/18.0.0/aem-groovy-console-all-18.0.0.zip)
```

Comment:
should we keep these versions inline? if so I guess the build should update it

Reply:
sure :D but wont do it in this PR though ^^

Reply:
acceptable xD